ddspipe_participants/src/cpp/participant/rtps/CommonParticipant.cpp

```diff
@@ -211,25 +211,16 @@ template<>
 DDSPIPE_PARTICIPANTS_DllAPI
 std::shared_ptr<eprosima::fastdds::rtps::UDPv4TransportDescriptor>
 CommonParticipant::create_descriptor(
-        std::set<types::IpType> whitelist)
+        std::set<types::WhitelistType> whitelist)
 {
     std::shared_ptr<eprosima::fastdds::rtps::UDPv4TransportDescriptor> udp_transport =
             std::make_shared<eprosima::fastdds::rtps::UDPv4TransportDescriptor>();
 
-    for (const types::IpType& ip : whitelist)
+    for (const types::WhitelistType& iface : whitelist)
     {
-        if (types::Address::is_ipv4_correct(ip))
-        {
-            udp_transport->interfaceWhiteList.emplace_back(ip);
-            EPROSIMA_LOG_INFO(DDSPIPE_COMMON_PARTICIPANT,
-                    "Adding " << ip << " to UDP whitelist interfaces.");
-        }
-        else
-        {
-            // Invalid address, continue with next one
-            EPROSIMA_LOG_WARNING(DDSPIPE_COMMON_PARTICIPANT,
-                    "Not valid IPv4. Discarding UDP whitelist interface " << ip << ".");
-        }
```

Comment:
Because now that interface names are accepted, it cannot be (easily) checked from our layer whether an element is an invalid IPv4 address (due to a typo for example) or an interface name. These checks are internally performed by Fast-DDS library, and their choice is that warnings are displayed if they fail (instead of throwing an exception or similar as we would prefer to have).